browserprocess/browser_process.go

```diff
@@ -1,4 +1,6 @@
-package common
+// Package browserprocess is in charge of the interaction
+// with the OS process that is running the browser.
+package browserprocess
```

Comment:
One suggestion: `browserprocess.BrowserProcess` stutters. Would it be better to rename it to `osext`? `osext.Process`?

That would also be consistent with the comment you wrote before the package definition: _"the interaction with the OS process"_. In the future, we can add more operating-system-related stuff into the same package without creating many packages. That is how it's done in the Go standard library as well. [See](https://pkg.go.dev/os).

Reply:
Yeah, nice idea. I've amended it in a1f72e472bcca9d4522a99892e4498da5111f771.

Reply:
Thanks for the amend! What about `BrowserProcess` to `Process`? I don't foresee any other process types in the near future :)

Reply:
I like that. It's confusing otherwise. Resolved in 8af5625586f2ccd7a1a2f29451840fe955076f24